clang/test/dpct/predefined_macro_replacement.cu

```diff
@@ -1,5 +1,8 @@
 // RUN: dpct --format-range=none --usm-level=none -out-root %T/predefined_macro_replacement %s --cuda-include-path="%cuda-path/include" -- -x cuda --cuda-host-only -D__NVCC__ -D__CUDACC__
 // RUN: FileCheck --input-file %T/predefined_macro_replacement/predefined_macro_replacement.dp.cpp --match-full-lines %s
+
+//CHECK: #define DPCT_COMPAT_RT_VERSION {{[1-9][0-9]+}}
```

Comment:
the value of the macro is defined accord to the header file used during migration?

Reply:
This macro definition will only appear in the file which used CUDART_VERSION or __CUDART_API_VERSION